cmd/maestro/server/api_server.go

```diff
@@ -118,6 +119,16 @@ func NewAPIServer(eventBroadcaster *event.EventBroadcaster) Server {
 	)(mainHandler)
 
 	mainHandler = removeTrailingSlash(mainHandler)
+	mainHandler = traceAttributeMiddleware(mainHandler)
+	if common.TracingEnabled() {
+		mainHandler = otelhttp.NewHandler(mainHandler, "apiserver",
```

Comment:
do we need this additional middleware since we also wrap in cmd/maestro/server/routes.go?

Reply:
@frzifus WDYT?

Reply:
I think one would be fine. In the ARO Frontend we we just merged additional attributes into the trace created by gorilla-mux.

Reply:
As we discussed, we will keep this code here as it is helpful to know the path of api call. We remove the mux code on routes for now.